tests/integration/cli/test_cli.py

```diff
@@ -208,8 +208,11 @@ def test_otx_e2e(
     assert latest_dir.exists()
 
     # 5) otx export with XAI
-    if "_cls" not in task or "dino" in model_name:
-        return
+    if ("_cls" not in task) and (task not in ["detection", "instance_segmentation"]):
+        pytest.skip("Supported only for classification, detection and instance segmentation task.")
```

Comment:
```suggestion
        return  # Supported only for classification, detection and instance segmentation task.
```
If pytest.skip exists here, model, which doesn't support XAI, is considered as 'skip' even if it passes everything except XAI.

Reply:
Thank you for the comment, fixed